sql/001-create-table-and-load-data.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS airports;
+
+CREATE TABLE airports(
+  airportCode VARCHAR(3) PRIMARY KEY,
+  airportName VARCHAR(16) NOT NULL,
+  country VARCHAR(16) NOT NULL
+);
+
+INSERT INTO airports (airportCode, airportName, country) VALUES('NRT' , ' NARITA' , 'JAPAN');
```

Comment:
NARITAの前の半角スペースはあえて入れていますか？
不要であれば消したいです。
```suggestion
INSERT INTO airports (airportCode, airportName, country) VALUES('NRT' , 'NARITA' , 'JAPAN');
```

Reply:
ご指摘いただきました箇所を修正しました。（置換機能は便利でした）
ef70af934bad3692d611ec55f1633c9968d920f2
![image](https://user-images.githubusercontent.com/113277395/203976978-7c1d387b-e07a-4562-9d33-9193db40378c.png)